### PR TITLE
Improve error message - CLI commands

### PR DESCRIPTION
**Pull request by krishnakalyan3**
## What does this PR do?

Addresses #14779

## Before submitting

- [x] Was this **discussed/approved** via a GitHub issue? (not for typos and docs)
- [x] Did you read the [contributor guideline](https://github.com/Lightning-AI/lightning/blob/master/.github/CONTRIBUTING.md), **Pull Request** section?
- [x] Did you make sure your **PR does only one thing**, instead of bundling different changes together?
- [x] Did you make sure to **update the documentation** with your changes? (if necessary)
- [ ] Did you write any **new necessary tests**? (not for typos and docs)
- [x] Did you verify new and **existing tests pass** locally with your changes?
- [ ] Did you list all the **breaking changes** introduced by this pull request?
- [ ] Did you **update the CHANGELOG**? (not for typos, docs, test updates, or minor internal changes/refactors)

## PR review

Anyone in the community is welcome to review the PR.
Before you start reviewing, make sure you have read the [review guidelines](https://github.com/Lightning-AI/lightning/wiki/Review-guidelines). In short, see the following bullet-list:

- [x] Is this pull request ready for review? (if not, please submit in draft mode)
- [x] Check that all items from **Before submitting** are resolved
- [x] Make sure the title is self-explanatory and the description concisely explains the PR
- [x] Add labels and milestones (and optionally projects) to the PR so it can be classified

## Did you have fun?

Make sure you had fun coding 🙃

### CODE REVIEW

File: src/lightning_app/cli/commands/connection.py [hunk 1]
```diff
@@ -31,7 +31,9 @@ def connect(app_name_or_id: str, yes: bool = False):
             if app_name_or_id == "localhost":
                 click.echo("You are connected to the local Lightning App.")
             else:
-                click.echo(f"You are already connected to the cloud Lightning App: {app_name_or_id}.")
+                click.echo(f"You are already connected to the cloud Lightning App: {app_name_or_id}."
+                            "Please run `lightning disconnect`"
```
Comment (Borda) [inline src/lightning_app/cli/commands/connection.py:35]:
```suggestion
                            " Please run `lightning disconnect`"
```

**Review by Borda — APPROVED** [review]

**Comment by krishnakalyan3** [comment]
cc: @tchaton

File: src/lightning_app/cli/commands/connection.py [hunk 2]
```diff
@@ -31,7 +31,10 @@ def connect(app_name_or_id: str, yes: bool = False):
             if app_name_or_id == "localhost":
                 click.echo("You are connected to the local Lightning App.")
             else:
-                click.echo(f"You are already connected to the cloud Lightning App: {app_name_or_id}.")
+                click.echo(
+                    f"You are already connected to the cloud Lightning App: {app_name_or_id}."
+                    " Please run `lightning disconnect`"
```
Comment (tchaton) [inline src/lightning_app/cli/commands/connection.py:36]:
```suggestion
                    " HINT: Use `lightning disconnect` to return back to the main Lighting CLI."
```

**Review by tchaton — APPROVED** [review]
LGTM !

**Review by hhsecond — APPROVED** [review]

**Review by manskx — APPROVED** [review]

**Review by lantiga — APPROVED** [review]

**Review by Borda — APPROVED** [review]

**Pull request closed without merging**